cloud/docker/docker.py

```diff
@@ -317,6 +324,7 @@
     - "Joshua Conner (@joshuaconner)"
     - "Pavel Antonov (@softzilla)"
     - "Ash Wilson (@smashwilson)"
+    - "David Dyball"
```

Comment:
If you want to be a maintainer that's fine.  Add your github id as well so we know who to ping :-)